SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -836,28 +836,31 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 
 		if (SpatialGDKSettings->UseRPCRingBuffer() && RPCService != nullptr)
 		{
-			EPushRPCResult Result = RPCService->PushRPC(TargetObjectRef.Entity, RPCInfo.Type, Payload);
+			const EPushRPCResult Result = RPCService->PushRPC(TargetObjectRef.Entity, RPCInfo.Type, Payload);
+			const EPushRPCResult Outcome = PushRPCResultUtils::GetOutcome(Result);
```

Comment:
Do we really need `PushRPCResultUtils::GetOutcome`? As I understand, it's supposed to remove invalid flags from `Result`? If we indeed need this function, maybe we can call it in `SpatialRPCService::PushRPC` to keep SpatialSender cleaner?

Reply:
It masks out the Outcome section of the bitmask, removing the Action section.

Reply:
SpatialSender currently needs both pieces of information for logging, but if we move the logging elsewhere we can move it